.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ jobs:
   # Run st2 Lint Checks
   lint:
     docker:
-      - image: circleci/python:2.7
+      - image: circleci/python:3.6
```

Comment:
FYI CircleCI is only for deb/rpm packaging and Unit/Integration test instructions in CircleCI are not used.
We'll need to remove them to avoid confusion, but that's another story.